packages/manager/modules/pci/src/projects/project/project.service.js

```diff
@@ -20,9 +20,45 @@ export default class {
       .catch(() => ({}));
   }
 
-  getCustomerRegions(serviceName) {
+  getCustomerRegions(serviceName, setHeader) {
```

Comment:
Perhaps we could use a more meaningful name like `withPaginationMode` ?
Also we could introduce some kind of options
```suggestion
  getCustomerRegions(serviceName, { withPaginationMode = false } = {}) {
```
```Javascript
Service.getCustomerRegions(serviceName, { withPaginationMode: true })
```
Versus
```Javascript
Service.getCustomerRegions(serviceName, true)
```

This allows two things :
- Just by reading the call code we know what the function expects and how it could be tweaked
- We can put more options later without having a long fixed list of arguments like `myMethod(true, undefined, 42, 'aString')` which is very hard to read / maintain